### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by ihjycc** [community/community#55976] [GitHub Education] [answered]
Hello everyone, I saw Congrats on the page! "Your request for @ihjycc was approved and your benefits will be available soon. "But I still haven't got the rewards, what should I do now?![image](https://github.com/community/community/assets/80345055/03b59786-6c79-4254-baf6-5b8240e0bd90)

**Comment by hardal7** [comment]
Hi ihjycc, you should be able to get your rewards within a day, please tag me to let me know if it takes more

**Reply by ihjycc** [reply]
hello, If I get a reward, what notification will I receive, I don't get any message now.

**Reply by hardal7** [reply]
What do you see when you go to this page?
https://education.github.com/pack/offers

**Reply by PradyumnaShuklaa** [reply]
i face the same issue and when i go there it says to me  that Are you a student?

**Reply by hardal7** [reply]
> i face the same issue and when i go there it says to me  that Are you a student?
>

Thanks for reaching out, can you send a screenshot please?

**Reply by PradyumnaShuklaa** [reply]
ok
sure

**Reply by ihjycc** [reply]
I saw "Hi, Huangjiayan! You were last verified as a student on May 20, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."
I want to know how long is the waiting period.

**Reply by ihjycc** [reply]
![image](https://github.com/community/community/assets/80345055/f4af1e75-110c-4f70-aa77-118310dbd7c9)

**Reply by hardal7** [reply, marked as answer]
Shouldn't take more than a week, you will get a mail sent from Github Education to you when you get access to the benefits.

**Comment by PradyumnaShuklaa** [comment]
![Capture](https://github.com/community/community/assets/85396659/bcf72682-c590-4a48-b428-6614e8db52d7)

**Reply by PradyumnaShuklaa** [reply]
@hardal7 check this screenshot
plz

**Reply by hardal7** [reply]
Did you try clicking: Yes I'm a student? What errors do you get?

**Reply by PradyumnaShuklaa** [reply]
yes i clicked
and i got redirected to-https://education.github.com/discount_requests/application

**Reply by hardal7** [reply]
Any screenshot of that page? Please be more spesific :)

**Reply by ihjycc** [reply]
![image](https://github.com/community/community/assets/80345055/47a33f2d-0d87-494f-a007-1d2a8f2f351d)

**Reply by ihjycc** [reply]
He told me "Congrats! Your request for @ihjycc was approved and your benefits will be available soon. "But I don't know how much longer

**Reply by PradyumnaShuklaa** [reply]
we have to wait ig for them to verify our academic affiliation for the final time

**Comment by rajatgoyal-ux** [comment]
same problem i am facing

**Reply by PradyumnaShuklaa** [reply]
my probelm fixed prob wait for 5 days

**Reply by Crysil25** [reply]
bro @hardal7 can you check my thread #58094 i was approved last june 11, 2023 but until now the benefits not release or available

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to receive your benefits. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
